namui/skia/src/render/image_filter.rs

```diff
@@ -1,14 +1,61 @@
 use crate::*;
+use std::fmt::Debug;
 
-#[derive(Debug, PartialEq, Clone, Hash, Eq)]
+#[derive(Debug, PartialEq, Clone, Hash, Eq, Default)]
 pub enum ImageFilter {
+    #[default]
+    Empty,
```

Comment:
Why do you need Empty?

Reply:
It makes many things easy. Compared to Option<ImageFilter> or ImageFilter::Image(blank_image)